packages/liveblocks-react/src/factory.tsx

```diff
@@ -585,7 +607,7 @@ export function createRoomContext<
     }
   }
 
-  function useSelector<T>(
+  function useStorageSelector<T>(
```

Comment:
should this be named `useStorage`? and we would also have `usePresenceSelector` as well? 

Or is the rational to keep `useStorage` for future utils which wouldn't work with `useStorageSelector`?

Reply:
Yeah, some naming cleanup needs to happen here. I renamed this one to distinguish it from `usePresence()` currently (which also is a selector-based API). The problem is that we cannot rename it to `useStorage()` just yet, because that hook already exists (in 0.17.9!) and some customers are using it, even though it's not documented officially. So we'll have to first deprecate that API to make room for that name… 🤔 

Definitely some polish to do here!